scipy/cluster/tests/test_hierarchy.py

```diff
@@ -1205,6 +1224,7 @@ def test_optimal_leaf_ordering(xp):
 
 
 @skip_xp_backends(np_only=True, reason='`Heap` only supports NumPy backend')
+@pytest.mark.uses_xp_capabilities(False, reason="private")
 def test_Heap(xp):
     values = xp.asarray([2, -1, 0, -1.5, 3])
```

Comment:
Maybe here is it cleaner just to remove the `xp` fixture?
```suggestion
def test_Heap():  # `Heap` only supports NumPy backend
    values = np.asarray([2, -1, 0, -1.5, 3])
```